reaktive/src/commonMain/kotlin/com/badoo/reaktive/completable/CompletableWrapper.kt

```diff
@@ -26,6 +26,28 @@ open class CompletableWrapper(inner: Completable) : Completable by inner {
             onError = onError,
             onComplete = onComplete
         )
+
+    @UseReturnValue
+    fun subscribe(
```

Comment:
I think it would be nice to include the subscribe option with no params passed.

Reply:
Good catch!

Reply:
Fixed